aptos-move/vm-genesis/src/lib.rs

```diff
@@ -741,17 +741,20 @@ fn initialize_keyless_accounts(session: &mut SessionExt, chain_id: ChainId) {
 }
 
 fn create_accounts(session: &mut SessionExt, accounts: &[AccountBalance]) {
-    let accounts_bytes = bcs::to_bytes(accounts).expect("AccountMaps can be serialized");
-    let mut serialized_values = serialize_values(&vec![MoveValue::Signer(CORE_CODE_ADDRESS)]);
-    serialized_values.push(accounts_bytes);
-    exec_function(
-        session,
-        GENESIS_MODULE_NAME,
-        "create_accounts",
-        vec![],
-        serialized_values,
-    );
-}
+    for account in accounts { // creating accounts one by one to avoid the quadratic complexity of the Move function create_accounts.
```

Comment:
@davidsupra , are we absolutely sure that all `accounts` here are unique? If so, this should be stated explicitly. Existing Move layer implementation doesn't make this assumption and checks for uniqueness (that is where the quadratic nature of this comes). If uniqueness is checked earlier, please add a comment that states that uniqueness is checked earlier and this would work with the precondition of uniqueness of `accounts`.

Reply:
We can ensure the uniqueness by changing the `accounts` parameter to a `BTreeSet`, which is what we actually use when we call the function. Can you please make this change Dr @davidsupra?

Reply:
I've made the change I suggested.